### DISCUSSION TITLE
catalog-mysql表字段日期&时间类型报错

### DISCUSSION THREAD

**Discussion by stevqin** [apache/doris#17098] [A - General / Q&A]
`CREATE EXTERNAL RESOURCE resource_ihr
properties (
    "type"="jdbc",
    "user" = "user",
    "password" = "pwd",
    "jdbc_url"="jdbc:mysql://10.1.16.145:3306/ihr_pro?useSSL=false&zeroDateTimeBehavior=convertToNull",
    "driver_url"="mysql-connector-java-5.1.47.jar",
    "driver_class"="com.mysql.jdbc.Driver"
);

CREATE CATALOG catalog_ihr WITH RESOURCE resource_ihr;
`
使用select语句报错
select * from catalog_ihr.ihr_pro.t_org_unit

`
[2023-02-24 12:47:41] [HY000][1105] errCode = 2, detailMessage = NoClassDefFoundError: com/mysql/jdbc/TimeUtil
`
若查询的表中字段 不是 日期、时间 类型的字段，则可以查询出将结果

--- Apache Doris 版本为 1.2.2
ps: mysql驱动试过 
mysql-connector-java-5.1.47.jar
mysql-connector-java-5.1.27-bin.jar
mysql-connector-java-8.0.25.jar
上面的错误依旧

**Comment by zy-kkk** [comment]
This is probably a known issue and will be fixed in the upcoming 1.2.3 release
